File: go/1480E-一维数组的动态和.go
/*
 * @Author: SukiEva
 * @Date: 2021-10-10 22:36:30
 * @Description: https://github.com/SukiEva
 * わたし、気になります！
 */

package main

func runningSum(nums []int) []int {
	for i := 1; i < len(nums); i++ {
		nums[i] += nums[i-1]
	}
	return nums
}

File: cpp/1480E-一维数组的动态和.cpp
/*
 * @Author: SukiEva
 * @Date: 2021-09-05 21:36:23
 * @Description: https://github.com/SukiEva
 * わたし、気になります！
 */

// https://leetcode-cn.com/problems/running-sum-of-1d-array/
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> runningSum(vector<int>& nums) {
        for (int i=1; i<nums.size(); i++){
            nums[i]+=nums[i-1];
        }
        return nums;
    }
};
